packages/streaming-image-volume-loader/src/BaseStreamingImageVolume.ts

```diff
@@ -107,6 +107,8 @@ export default class BaseStreamingImageVolume extends ImageVolume {
       windowCenter,
       windowWidth,
       color,
+      // we don't use rgb for the volume actors, so this is always false
```

Comment:
But this is saying rgba is false, not rgb is false, and I thought the point of the entire exercise was to be able to display color volumes?  Could you make the comment more descriptive.